modules/accredited_representative_portal/app/services/accredited_representative_portal/saved_claim_service/create.rb

```diff
@@ -50,37 +42,46 @@ def perform(
 
         ##
         # Expose a discrete set of known exceptions. Expose any remaining with a
-        # catch-all exception.
+        # catch-all unknown exception.
         #
-        rescue WrongAttachmentsError
+        rescue RecordInvalidError, WrongAttachmentsError
```

Comment:
I think you may want to make sure error messages provide enough context to diagnose the errors